src/main/scala/org/ergoplatform/Input.scala

```diff
@@ -44,14 +37,16 @@ case class Input(override val boxId: BoxId, spendingProof: ProverResult)
 
 object Input {
   object serializer extends Serializer[Input, Input] {
```

Comment:
@greenhat  Consider using in serialization of boxes.

Reply:
@ergomorphic I did not get that. Could you please clarify?